conf/application.conf

```diff
@@ -14,9 +14,17 @@
 
 # This is the main configuration file for the application.
 # ~~~~~
+include "frontend.conf"
 
 appName=hmrc-email-renderer
 
+play.http.router=app.Routes
```

Comment:
This file contains `play.http.router=app.Routes` and also `application.router=prod.Routes`. As `prod.Routes` includes `app.Routes` I think you only need `prod.Routes` and I think you should only use `play.http.router` as `application.router` is deprecated.